src/ticketing/utils.ts

```diff
@@ -178,6 +178,7 @@ export function isCanBeConsumedNowFareContract(
   currentUserId: string | undefined,
 ) {
   if (f.customerAccountId !== currentUserId) return false;
+  if (f.state !== 1) return false;
```

Comment:
Could you use the `FareContractState` enum instead? That makes this more readable.

Reply:
I totally forgot I could do that... updated 🤦